mautic/mautic
# Segment public name property for preference center

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new segment property - public name which is visible in preference center. Includes migration that sets Public name with same value as Name. Public name value is not required and 

![image](https://user-images.githubusercontent.com/8580942/70930625-e83eb280-2035-11ea-9dd5-646e98ec52b6.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Run `php app/console doctrine:migrations:migrate`
3. Create segment with different Name and Public name
4. Activate segment Available in Preference Center option 
5. Activate preferences center, send mail with unsubscribe link
6. Open contact preferences center (click unsubscribe link)
7. In preferences center should appear public segment name
